app/scripts/utils/show-mouse-position.js

```diff
@@ -75,7 +79,7 @@ const showMousePosition = (
    * @param  {Object}  e  Event object.
    */
   const mouseMoveHandler = (event) => {
-    if (!event.hoveredTracks.length) {
+    if (event.noHoveredTracks) {
```

Comment:
To me the alternative (!event.hoveredTracks.length) is more versatile because we don't have to keep track of both `event.hoveredTracks` and `event.noHoveredTrack`. I think the text "no hovered tracks" is helpful but as a comment rather than as a variable.

Reply:
I agree but the reason why I had to change this is because `hoveredTracks` is no longer available. This due to the fact that `hoveredTracks` is referencing track objects which are not available in another window. Therefore, the globally published `higlass.mouseMove` event does not have the property `hoveredTracks` while the locally does. An alternative would be to always publish an empty array globally but I think this would give the illusion that one could somehow get an event with a non-empty `hoveredTracks` property which is impossible. Does that make sense? 🤔🙂